dotnet/src/webdriver/Response.cs

```diff
@@ -158,24 +158,24 @@ public static Response FromErrorJson(string value)
 
             if (!deserializedResponse.TryGetValue("value", out var valueObject))
             {
-                throw new WebDriverException($"The 'value' property was not found in the response:{Environment.NewLine}{value}");
+                throw new WebDriverException($"The 'value' property was not found in the response:{System.Environment.NewLine}{value}");
```

Comment:
All changes in this file looks unnecessary.

Reply:
I had some compilation errors before, but I can't reproduce them anymore. Reverting.